<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>603</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="Background">
      <property name="styleSheet">
       <string notr="true">#Background{
background-color: rgb(19, 19, 19);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QFrame" name="Header">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">*{
color: rgb(255, 255, 255);
background:black;
border:none;
}
QPushButton{
color: rgb(255, 255, 255);
background:transparent;
border:none;
}

QPushButton:hover{
background:white;
border:none;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton:hover{
	background-color: rgb(255, 0, 0);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../PyQt/Icons/white/x.svg</normaloff>../PyQt/Icons/white/x.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_2:hover{
	
	background-color: rgb(150, 150, 150);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../PyQt/Icons/white/maximize.svg</normaloff>../PyQt/Icons/white/maximize.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_3:hover{
	
	background-color: rgb(181, 181, 181);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../PyQt/Icons/white/minus.svg</normaloff>../PyQt/Icons/white/minus.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(85, 255, 127);</string>
               </property>
               <property name="text">
                <string>OFFLINE SPEECH TO TEXT</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_4:hover{
	
	background-color: rgb(150, 150, 150);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../PyQt/Icons/white/alert-circle.svg</normaloff>../PyQt/Icons/white/alert-circle.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>35</width>
                 <height>35</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Main">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>91</width>
            <height>551</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(10, 10, 10);
color: rgb(255, 255, 255);
border:none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QToolButton" name="toolButton_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>240</y>
             <width>71</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PyQt/Icons/white/settings.svg</normaloff>../PyQt/Icons/white/settings.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>330</y>
             <width>71</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../PyQt/Icons/white/help-circle.svg</normaloff>../PyQt/Icons/white/help-circle.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>61</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton_5{
	background-color: rgb(38, 116, 0);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#pushButton_5:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgb(38, 116, 0);
background-position:calc(100%-10px)center;
}
QPushButton#pushButton_5:hover{
background-color: rgb(38, 116, 0);
}</string>
           </property>
           <property name="text">
            <string>START</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>61</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton_6{
	background-color: rgb(189, 0, 0);
color:rgba(255,255,255,200);
border-radius:5px;
}
QPushButton#pushButton_6:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgb(189, 0, 0);
background-position:calc(100%-10px)center;
}
QPushButton#pushButton_6:hover{
background-color: rgb(189, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>STOP</string>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="frame_5">
          <property name="geometry">
           <rect>
            <x>115</x>
            <y>11</y>
            <width>381</width>
            <height>251</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(85, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6"/>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>10</y>
            <width>801</width>
            <height>331</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;
color: rgb(78, 234, 234);
border: 2px solid rgb(170, 255, 127);
border-radius:10px;</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
         <widget class="QFrame" name="frame_6">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>370</y>
            <width>801</width>
            <height>151</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid rgb(170, 0, 255);
border-radius:10px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="pushButton_7">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>80</y>
             <width>61</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton_7{
	background-color: rgb(222, 74, 222);
	color: rgb(0, 0, 0);
border-radius:5px;
border: rgb(222, 74, 222);
}
QPushButton#pushButton_7:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgb(222, 74, 222);
background-position:calc(100%-10px)center;
}
QPushButton#pushButton_7:hover{
background-color: rgb(222, 74, 222);
}</string>
           </property>
           <property name="text">
            <string>Upload</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_8">
           <property name="geometry">
            <rect>
             <x>720</x>
             <y>80</y>
             <width>61</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#pushButton_8{
	background-color: rgb(206, 206, 0);
	color: rgb(0, 0, 0);
border-radius:5px;
border: rgb(236, 236, 117)
}
QPushButton#pushButton_8:pressed{
padding-left:5px;
padding-top:5px;
background-color: rgb(206, 206, 0);
background-position:calc(100%-10px)center;
}
QPushButton#pushButton_8:hover{
background-color: rgb(206, 206, 0);
}</string>
           </property>
           <property name="text">
            <string>Convert</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>20</y>
             <width>351</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
border: none</string>
           </property>
           <property name="text">
            <string>CONVERT AUDIO FILE TO TEXT</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>80</y>
             <width>531</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
border:2px solid rgb(163, 163, 163);</string>
           </property>
           <property name="text">
            <string>                                                    No File Selected !!!</string>
           </property>
          </widget>
         </widget>
         <zorder>frame_5</zorder>
         <zorder>frame_4</zorder>
         <zorder>plainTextEdit</zorder>
         <zorder>frame_6</zorder>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="Footer">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images/res.qrc"/>
 </resources>
 <connections/>
</ui>
